MANIFEST.in

```diff
@@ -11,6 +11,12 @@ prune answer-store
 recursive-include yt *.py *.pyx *.pxi *.pxd *.h *.hpp README* *.txt LICENSE* *.cu
 recursive-include doc *.rst *.txt *.py *.ipynb *.png *.jpg *.css *.html
 recursive-include doc *.h *.c *.sh *.svgz *.pdf *.svg *.pyx
+recursive-exclude yt *.cpp *.c
```

Comment:
should the `*.c` be removed from the `recursive-include` line just above this?

Reply:
That line is for files under `docs`. Theres maybe one or two c files in there and I don't think it hurts to keep distributing them.

Reply:
oh right, read too fast and missed that `doc` there :)